### PR TITLE
Adds view-level unit specs for `Marketplace::Cart::DeliveryAreaComponent'

### PR DESCRIPTION
**Pull request by rosschapman**
Follow-up to https://github.com/zinc-collective/convene/pull/1920

### CODE REVIEW

File: app/furniture/marketplace/cart/delivery_area_component.html.erb [hunk 1]
```diff
@@ -6,6 +6,9 @@
         <%= delivery_area.label %>
         <div class="text-sm text-gray-500">
           <%= render(Marketplace::Cart::DeliveryExpectationsComponent.new(cart: cart)) %>
+          <%- if single_delivery_area? %>
+            <div><small>Orders outside of this location will be subject to cancellation.</small></div>
+          <% end %>
```
Comment (rosschapman) [inline app/furniture/marketplace/cart/delivery_area_component.html.erb:11]: Testing finds (UX) bugs. 💪

Comment (zspencer) [reply app/furniture/marketplace/cart/delivery_area_component.html.erb:11]: Eh, it looks good enough for now :)

File: spec/furniture/marketplace/cart/delivery_area_component_spec.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+RSpec.describe Marketplace::Cart::DeliveryAreaComponent, type: :component do
+  subject(:output) { render_inline(component) }
+
+  let(:component) { described_class.new(cart: cart) }
+
+  let(:cart) { create(:marketplace_cart, :with_products, marketplace: marketplace) }
```
Comment (zspencer) [inline spec/furniture/marketplace/cart/delivery_area_component_spec.rb:8]: Why does the cart need products?

**Review by zspencer — APPROVED** [review]
Yay! Tests! Thank you!

File: spec/furniture/marketplace/cart/delivery_area_component_spec.rb (hunk 2 above)
Comment (rosschapman) [reply spec/furniture/marketplace/cart/delivery_area_component_spec.rb:8]: Ah good catch. I got confused for a moment and thought I needed this to display stuff (that was outside the component scope, actually) and forgot to remove it.

**Pull request merged**